### PR TITLE
🚀 Spell check all md files

### PR DESCRIPTION
**Pull request by YazeedAlKhalaf**

### CODE REVIEW

**Comment by subfuzion** [comment]
Hi @YazeedAlKhalaf  -- thanks for contributing! We have very stringent markdown lint checks in place, which is why the build is failing for this PR. You can run markdown lint checks on your machine before pushing an update to your PR by installing markdownlint-cli2 and running a check from the project root directory, like this:

```
npm install --global  markdownlint-cli2
markdownlint-cli2 "**/*.md"   # or if you don't want to let it try to auto-fix:  markdownlint-cli2-fix "**/*.md"
```

Be wary of changing reference links at the bottom of markdown docs. They are split across two lines because of the max line length rule.

**Comment by YazeedAlKhalaf** [comment]
> Hi @YazeedAlKhalaf -- thanks for contributing! We have very stringent markdown lint checks in place, which is why the build is failing for this PR. You can run markdown lint checks on your machine before pushing an update to your PR by installing markdownlint-cli2 and running a check from the project root directory, like this:
> 
> ```
> npm install --global  markdownlint-cli2
> markdownlint-cli2 "**/*.md"   # or if you don't want to let it try to auto-fix:  markdownlint-cli2-fix "**/*.md"
> ```
> 
> Be wary of changing reference links at the bottom of markdown docs. They are split across two lines because of the max line length rule.

Ah okay will fix them right away 🚀

File: README.md [hunk 1]
```diff
@@ -93,37 +93,35 @@ properly format your code.
...
+
 [ff_dart]: https://github.com/GoogleCloudPlatform/functions-framework-dart
 
 <!-- Unit Test links -->
-[ff_dart_unit_img]:
-https://github.com/GoogleCloudPlatform/functions-framework-dart/workflows/Dart%20Unit%20CI/badge.svg
-[ff_dart_unit_link]:
-https://github.com/GoogleCloudPlatform/functions-framework-dart/actions?query=workflow%3A"Dart+Unit+CI"
+
+[ff_dart_unit_img]: https://github.com/GoogleCloudPlatform/functions-framework-dart/workflows/Dart%20Unit%20CI/badge.svg
```
Comment (subfuzion) [inline README.md:101]: Due to the space after the colon, this fails the line length limit. When we split across two lines, it's ok that the URL itself is too long (won't cause a lint error).

File: docs/01-introduction.md [hunk 2]
```diff
@@ -61,7 +61,7 @@ Dart tests, which we'll cover as part of this quickstart.
 
 ## What's next
 
-Using the example presented above, we will walk through a few quickstart
+Using the example presented above, we will walk through a few quickstarts
```
Comment (subfuzion) [inline docs/01-introduction.md:64]: Double plurals (quickstarts scenarios) doesn't read well.

**Review by subfuzion — CHANGES_REQUESTED** [review]
This is a bit more than a "spell check" PR, will take a bit more time to review. In the meantime, please look at comments and rerun `markdownlint-cli2`. Thanks!

File: README.md [hunk 3]
```diff
@@ -8,11 +8,11 @@
...
 
-|Functions Framework|Unit Tests|Lint Test|Conformance Tests|
-|---|---|---|---|
-[Dart][ff_dart]| [![][ff_dart_unit_img]][ff_dart_unit_link] | [![][ff_dart_lint_img]][ff_dart_lint_link] | [![][ff_dart_conformance_img]][ff_dart_conformance_link] |
+| Functions Framework | Unit Tests                                 | Lint Test                                  | Conformance Tests                                        |
+| ------------------- | ------------------------------------------ | ------------------------------------------ | -------------------------------------------------------- |
+| [Dart][ff_dart]     | [![][ff_dart_unit_img]][ff_dart_unit_link] | [![][ff_dart_lint_img]][ff_dart_lint_link] | [![][ff_dart_conformance_img]][ff_dart_conformance_link] |
 
-An open source FaaS (Function as a Service) framework for writing portable Dart
+An open-source FaaS (Function as a Service) framework for writing portable Dart
```
[the remark this answers is not in the document]
Comment (YazeedAlKhalaf) [reply README.md:15]: Ah okay, new information 👌

File: README.md (hunk 1 above)
Comment (YazeedAlKhalaf) [reply README.md:101]: Done 🚀
all needed splits are done with now extra trailing space and the linter returned 0 errors!

File: docs/01-introduction.md (hunk 2 above)
Comment (YazeedAlKhalaf) [reply docs/01-introduction.md:64]: Fixed! 🐛🚀

**Comment by subfuzion** [comment]
@YazeedAlKhalaf The build is failing because you modified `dartfn/templates/cloudevent/README.md`. This requires rebuilding the associated [generated template](https://github.com/GoogleCloudPlatform/functions-framework-dart/blob/main/dartfn/lib/src/generators/cloudevent.g.dart). Unfortunately, you mentioned [here](#151) that `build_runner` is not working for you. I recommend undoing the changes to this file and resubmitting. When this PR is closed, you can open an issue to synchronize whatever changes were made to [examples/raw_cloudevent/README.md](https://github.com/GoogleCloudPlatform/functions-framework-dart/tree/main/examples/raw_cloudevent) with the template and then I can take care of that.

**Comment by subfuzion** [comment]
Hi @YazeedAlKhalaf, thanks for the PR. Why don't you reach out to me directly and I'll work with you to help you get tests passing for you. Since this PR has gotten stale, let's close it for now. Feel free to [email me](mailto:tonypujals@google.com) and I'll schedule a meeting with you.

**Pull request closed without merging**